content/ec2-auto-scaling-with-multiple-instance-types-and-purchase-options/spot_resilience.md

```diff
@@ -0,0 +1,95 @@
+      "account": "123456789012",
+      "time": "2019-11-05T11:03:11Z",
+      "region": "eu-west-1",
+      "resources": [
+        "arn:aws:ec2:eu-west-1b:instance/<instance-id>"
+      ],
+    "detail": {
+      "instance-id": "<instance-id>",
+      "instance-action": "terminate"
+      }
+    }
+    ```
+    
+  1. Replace both occurrences of **"\<instance-id>"** with the instance-id of one of the Spot Instances that are currently running in your EC2 Auto Scaling group (you can get an instance-id from the Instances tab in the bottom pane of the [EC2 Auto Scaling groups console](https://console.aws.amazon.com/ec2/autoscaling/home#AutoScalingGroups:view=details) ). You don't need to change any of the other parameters in the event json.
+  1. Click **Create**
+  1. With your new test name (i.e TestSpotInterruption) selected in the dropdown menu, click the **Test** button.
+  1. The execution result should be **succeeded** and you can expand the details to see the successful log message: "Instance i-01234567890123456 belongs to AutoScaling Group runningAmazonEC2WorkloadsAtScale. Detaching instance..."
+  1. Go back to the [EC2 Auto Scaling groups console](https://console.aws.amazon.com/ec2/autoscaling/home#AutoScalingGroups:view=details), and under the **Activity History** tab in the bottom pane, you should see a **Detaching EC2 instance** activity, followed shortly after by a **Launching a new EC2 instance** activity.
+  1. Go to the [EC2 ELB Target Groups console](https://console.aws.amazon.com/ec2/v2/home?1#TargetGroups:sort=targetGroupName) and click on the **runningAmazonEC2WorkloadsAtScale** Target Group, go to the Targets tab in the bottom pane, you should see the instance in `draining` mode.
+
+Great result! by leveraging the EC2 Spot Instance Interruption Warning, the Lambda Function detached the instance from the Auto Scaling group and the ELB Target Group, thus draining existing connections, and launching a replacement instance before the current instance is terminated.
+
+{{% notice warning %}} 
+In a real scenario, EC2 would terminate the instance after two minutes, however in this case we simply mocked up the interruption so the EC2 instance will keep running outside the Auto Scaling group. Go to the EC2 console and terminate the instance that you used on the mock up event.
+{{% /notice %}}
+
+
+### Increasing the application's resilience when using Spot Instances
+
+In a previous step in this workshop, you learned that the EC2 Auto Scaling group is configured to fulfill the 4 lowest-priced instance types (out of a list of 9 types) in each Availability Zone. Since Spot is spare EC2 capacity, its supply and demand vary. By diversifying your usage of capacity pools (a combination of an instance type in an Availability Zone), you increase your chances of getting the desired capacity, and decrease the potential number of interrupted instances in case Spot Instances are interrupted (when EC2 needs the capacity back for On-Demand).
```

Comment:
I'd suggest reviewing the number and use of parenthesis; Is there any reason we shouldn't be using `,`